app/services/notices/setup/prepare-return-forms.service.js

```diff
@@ -12,28 +12,23 @@ const PrepareReturnFormsPresenter = require('../../../presenters/notices/setup/p
 /**
  * Orchestrates fetching and presenting the data for the return form
  *
- * @param {module:SessionModel} session - The session instance
- * @param {string} returnId - The UUID of the return log
+ * We return the 'pageData' to be used when sending the notification. The legacy code relies on setting the
+ * personalisation in the database (mainly the address and due dates).
+ *
+ * @param {string} licenceRef
+ * @param {object} dueReturnLog
```

Comment:
We need some documentation for these params, plus the `recipient` for the JSDoc to be complete.